src/app/util/af.h

```diff
@@ -184,6 +184,46 @@ uint16_t emberAfIndexFromEndpointIncludingDisabledEndpoints(chip::EndpointId end
  */
 uint16_t emberAfFindClusterServerEndpointIndex(chip::EndpointId endpoint, chip::ClusterId clusterId);
 
+/**
+ *  @brief Returns an index of the given endpoint the given cluster server
+ *
+ * Returns kEmberInvalidEndpointIndex if the given endpoint does not support the
+ * given cluster or if the given endpoint is disabled.
+ *
+ * Unlike emberAfFindClusterServerEndpointIndex, this function always returns the same index
+ * for a given endpointId instance, static or dynamic.
+ *
+ * The return index is determined by the location of the endpoint in
+ * emAfEndpoints database but is adjusted to the given Cluster server implementation scope
```

Comment:
I don't know what this comment is supposed to say.  Maybe better would be:
```suggestion
 * The return index is identical to emberAfFindClusterServerEndpointIndex for fixed endpoints,
 * but for dynamic endpoints the indexing assumes that any dynamic endpoint could start supporting
 * the given server cluster.
```
or something like that.